addons/test_website_slides_full/static/tests/tours/slides_certification_member.js

```diff
@@ -50,16 +49,22 @@ var buyCertificationSteps = [{
     run: "click",
 },
     tourUtils.goToCart(),
-    tourUtils.goToCheckout(),
-    ...tourUtils.payWithDemo(),
-    wTourUtils.clickOnExtraMenuItem({}),
 {
-    content: 'eCommerce: go back to e-learning home page',
-    trigger: '.nav-item a:contains("Courses")',
-    run: "click",
+    content: 'Payment with Demo',
+    trigger: 'button:contains("Pay with Demo")',
+    run: 'click',
+}, {
+    content: 'eCommerce: add card number',
+    trigger: 'input[name="customer_input"]',
+    run: "edit 4343434343434343",
+},
+    tourUtils.pay(),
+{
+    content: 'eCommerce: check that the payment is successful',
+    trigger: '.oe_website_sale_tx_status:contains("Your payment has been successfully processed.")',
```

Comment:
Why do we need to write this code again?
we have the function?

Reply:
`payWithDemo` function only works on `/shop/payment` page, and we're not going there as we have option to pay directly from `/shop/cart`.